src/native_main.nim

```diff
@@ -37,6 +37,36 @@ proc sanitiseFilename(fn: string): string =
 
     return replace(ans,"..",".")
 
+# Split command line in a list honouring quotes (and also insert "/c" in the
+# beginning). Only used under Windows when shelling out to cmd.exe.
+#
+# This is extremely simplistic and doesn't handle escaping quotes at all.
+proc buildCmdArgs(cmd: string): seq[string] =
+    var args: seq[string]
+    args.add("/c")
+    var arg: string
+    var inQuotes: bool = false
+    for c in cmd:
+        case c:
+            of ' ':
+                if inQuotes:
+                    arg.add(c)
+                else:
+                    if arg != "":
+                        args.add(arg)
+                        arg = ""
+
+            of '"':
```

Comment:
```suggestion
            of '"', '\'':
```

I think cmd treats the two interchangeably?

Reply:
No, `'` is not special to it. Just try running `gvim.exe -c "set ft=markdown"` and `gvim.exe -c 'set ft=markdown'` from its prompt.

Reply:
I don't think it does – as far as I understand, single quotes are treated like normal characters: see [this SO answer](https://stackoverflow.com/a/51090074/9859881).

Reply:
> No, ' is not special to it.

Thanks : )

> Just try running gvim.exe -c "set ft=markdown" and gvim.exe -c 'set ft=markdown' from its prompt

If I had access to a Windows machine I probably would have fixed this myself :P

Reply:
FWIW you can use Wine for simple tests like this (but you probably don't want to run FF under Wine...).